src/main/java/uk/gov/justice/digital/delius/controller/secure/AuthenticationController.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import io.swagger.annotations.*;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.justice.digital.delius.controller.advice.ErrorResponse;
+import uk.gov.justice.digital.delius.data.api.UserDetails;
+import uk.gov.justice.digital.delius.service.UserService;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+@RequestMapping(value = "secure", produces = MediaType.APPLICATION_JSON_VALUE)
+@Slf4j
+@Api(tags = "Authentication", authorizations = {@Authorization("ROLE_AUTH_DELIUS_LDAP")}, description = "Authentication for Delius Identity (LDAP)")
+@AllArgsConstructor
+@PreAuthorize("hasRole('ROLE_AUTH_DELIUS_LDAP')")
+public class AuthenticationController {
+
+    private final UserService userService;
+
+    @ApiOperation(
+            value = "Authenticate a username and password against Delius Identity (LDAP)",
+            authorizations = {@Authorization("ROLE_AUTH_DELIUS_LDAP")})
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 200, message = "OK"),
+                    @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+                    @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class)
+            })
+    @GetMapping("/authenticate")
```

Comment:
Should this be a POST instead?

Reply:
Definitely.  If you have it as a `GET` then the username and password will appear in app insights